<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <title>Publications - Weiwei Duan | 段伟玮</title>
  <meta name="description" content="Weiwei Duan's publications.">
  <meta name="author" content="Weiwei Duan | 段伟玮">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" type="text/css">
  <link rel="icon" type="image/png" href="../images/favicon.png">
  <style>.nanx-footer{font-size:90%;}.nanx-footer a{font-weight:500;color:#495057}.nanx-footer a:hover{color:#007bff}</style>
  <style>author {font-weight: bold; text-decoration: underline;}</style>
</head>

<body>

<div class="pt-5">
  <div class="container">
    <div class="row">

      <div class="col-md-10 offset-1">
        <div class="row">

          <div class="col-md-2">
            <img src="../images/Weiwei_Duan_img.jpg" class="img-fluid d-block rounded">
          </div>

          <div class="col-md-9">
            <h1 class="display-4">Weiwei Duan</h1>
<!--             <p class="text-muted">Senior Data Scientist. Seven Bridges Genomics, Inc. Cambridge, MA, USA.<br>
              PhD Candidate. Statistics. Central South University. Changsha, Hunan, China.</p> -->
          </div>

        </div>
      </div>

    </div>
  </div>
</div>

<div class="">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <hr>
        <ul class="nav nav-pills nav-justified">
          <li class="nav-item">
            <a href="/" class="nav-link">Home</a>
          </li>
          <li class="nav-item">
            <a href="#" class="nav-link">Publications</a>
          </li>
          <li class="nav-item">
            <a href="/projects/" class="nav-link">Projects</a>
          </li>
          <li class="nav-item">
            <a href="/cv/" class="nav-link">Curriculum Vitae</a>
          </li>
        </ul>
        <hr>
      </div>
    </div>
  </div>
</div>

<a class="anchor" id="top"></a>

<!-- <div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">Research Profiles</h2>

        <p></p>

        <p>
          <a href="https://scholar.google.com/citations?user=BNizRecAAAAJ&hl=en">Google Scholar</a>
          &middot; <a href="https://www.researchgate.net/profile/Nan_Xiao3">ResearchGate</a>
          &middot; <a href="https://www.mendeley.com/profiles/nan-xiao/">Mendeley</a>
          &middot; <a href="https://profiles.impactstory.org/u/0000-0002-0250-5673">Impactstory</a>
          &middot; <a href="https://orcid.org/0000-0002-0250-5673">ORCID</a>
          &middot; <a href="http://www.researcherid.com/rid/K-7326-2016">ResearcherID</a>
        </p>

      </div>
    </div>
  </div>
</div> -->

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">Preprints</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <p>
          <author>Nan Xiao</author>, Q.-S. Xu, and M.-Z. Li.
          <a href="https://www.biorxiv.org/content/biorxiv/early/2016/08/23/065524.full.pdf">
            hdnom: building nomograms for penalized Cox models with high-dimensional survival data</a>.
          <em>bioRxiv</em>.
          doi:<a href="https://doi.org/10.1101/065524">10.1101/065524</a>.
          <a class="codelink" href="#hdnom">abstract</a> |
          <a class="codelink" href="https://nanx.me/hdnom/">software</a>
        </p>
      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">2018</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <p>
          L.-L. Wang, Y.-W. Lin, X.-F. Wang, <author>Nan Xiao</author>, Y.-D. Xu, H.-D. Li, Q.-S. Xu.
          <a href="interval-variable-selection.pdf">
            A selective review and comparison for interval variable selection in spectroscopic modeling</a>.
          <em>Chemometrics and Intelligent Laboratory Systems</em> 172: 229-240, 2018.
          doi: <a href="https://doi.org/10.1016/j.chemolab.2017.11.008">10.1016/j.chemolab.2017.11.008</a>.
          <a class="codelink" href="#ivs">abstract</a>
        </p>
      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">2017</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <p>
          Y.-W. Lin, <author>Nan Xiao</author>, L.-L. Wang, C.-Q. Li, Q.-S. Xu.
          <a href="OHPL.pdf">
            Ordered homogeneity pursuit lasso for group variable selection with applications to spectroscopic data</a>.
          <em>Chemometrics and Intelligent Laboratory Systems</em> 168: 62-71, 2017.
          doi: <a href="https://doi.org/10.1016/j.chemolab.2017.07.004">10.1016/j.chemolab.2017.07.004</a>.
          <a class="codelink" href="#ohpl">abstract</a> |
          <a class="codelink" href="https://OHPL.io/">website</a> |
          <a class="codelink" href="https://OHPL.io/doc/">software</a>
        </p>
      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">2016</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <p>
          L. Shen, D.-S. Cao, Q.-S. Xu, X. Huang, <author>Nan Xiao</author>, Y.-Z. Liang.
          <a href="MRKNN.pdf">
            A novel local manifold-ranking based <em>k</em>-NN for modeling the regression between bioactivity and molecular descriptors</a>.
          <em>Chemometrics and Intelligent Laboratory Systems</em> 151: 71-77, 2016.
          doi: <a href="https://doi.org/10.1016/j.chemolab.2015.12.005">10.1016/j.chemolab.2015.12.005</a>.
          <a class="codelink" href="#mrknn">abstract</a>
        </p>
      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">2015</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <p>
          <author>Nan Xiao</author> and Q.-S. Xu.
          <a href="msaenet.pdf">
            Multi-step adaptive elastic-net: reducing false positives in high-dimensional variable selection</a>.
          <em>Journal of Statistical Computation and Simulation</em> 85(18): 3755-3765, 2015.
          doi: <a href="https://doi.org/10.1080/00949655.2015.1016944">10.1080/00949655.2015.1016944</a>.
          <a class="codelink" href="#msaenet">abstract</a> |
          <a class="codelink" href="https://nanx.me/msaenet/">software</a>
        </p>

        <p>
          <author>Nan Xiao</author>, D.-S. Cao, M.-F. Zhu, and Q.-S. Xu.
          <a href="protr.pdf">
            protr/ProtrWeb: R package and web server for generating various numerical representation schemes of protein sequence</a>.
          <em>Bioinformatics</em> 31(11): 1857-1859, 2015.
          doi: <a href="https://doi.org/10.1093/bioinformatics/btv042">10.1093/bioinformatics/btv042</a>.
          <a class="codelink" href="#protr">abstract</a> |
          <a class="codelink" href="https://nanx.me/protr/">software</a>
        </p>

        <p>
          D.-S. Cao*, <author>Nan Xiao*</author>, Q.-S. Xu and A. F. Chen.
          <a href="Rcpi.pdf">
            Rcpi: R/Bioconductor package to generate various descriptors of proteins, compounds, and their interactions</a>.
          <em>Bioinformatics</em> 31(2): 279-281, 2015.
          doi: <a href="https://doi.org/10.1093/bioinformatics/btu624">10.1093/bioinformatics/btu624</a>.
          *Joint first authors.
          <a class="codelink" href="#Rcpi">abstract</a> |
          <a class="codelink" href="https://nanx.me/Rcpi/">software</a>
        </p>

        <p>
          D.-S. Cao, <author>Nan Xiao</author>, Y.-J. Li, W.-B. Zeng, Y.-Z. Liang, A.-P. Lu, Q.-S. Xu, A. F. Chen.
          <a href="MEF.pdf">
            Integrating multiple evidence sources to predict adverse drug reactions based on systems pharmacology model</a>.
          <em>CPT: Pharmacometrics & Systems Pharmacology</em> 4(9): 498–506, 2015.
          doi: <a href="https://doi.org/10.1002/psp4.12002">10.1002/psp4.12002</a>.
          <a class="codelink" href="#MEF">abstract</a> |
          <a class="codelink" href="https://github.com/road2stat/MEF">code and data to reproduce results from paper</a>
        </p>

        <p>
          J.-B. Wang, D.-S. Cao, M.-F. Zhu, Y.-H. Yun, <author>Nan Xiao</author>, Y.-Z. Liang.
          <a href="logd.pdf">
            <em>In silico</em> evaluation of logD<sub>7.4</sub> and comparison with other prediction methods</a>.
          <em>Journal of Chemometrics</em> 29(7): 389-398, 2015.
          doi: <a href="https://doi.org/10.1002/cem.2718">10.1002/cem.2718</a>.
          <a class="codelink" href="#logd">abstract</a> |
          <a class="codelink" href="https://github.com/road2stat/logd74">dataset</a>
        </p>
      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<div class="pt-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <h2 class="">Abstracts</h2>
        <hr>
      </div>
    </div>
  </div>
</div>

<a class="anchor" id="hdnom"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>hdnom: Building Nomograms for Penalized Cox Models with High-Dimensional Survival Data</h3>

        <p>
          <author>Nan Xiao</author>, Q.-S. Xu, and M.-Z. Li.
        </p>

        <p>
          <img src="../images/paper-hdnom.png" class="img-fluid rounded mx-auto d-block" alt="hdnom" width="100%"/>
        </p>

        <p>
          <strong>Abstract:</strong>
        </p>
        <p><strong>Summary:</strong> We developed hdnom, an R package for survival modeling
          with high-dimensional data. The package is the first free and open-source software package
          that streamlines the workflow of penalized Cox model building, validation, calibration,
          comparison, and nomogram visualization, with nine types of penalized Cox regression methods
          fully supported. A web application and an online prediction tool maker are offered to
          enhance interactivity and flexibility in high-dimensional survival analysis.</p>
        <p> <strong>Availability:</strong> The hdnom R package is available from CRAN:
          <a href="https://cran.r-project.org/package=hdnom" target="_blank">https://cran.r-project.org/package=hdnom</a> under GPL.
          The hdnom web application can be accessed at <a href="http://hdnom.io" target="_blank">http://hdnom.io</a>.
          The web application maker is available from <a href="https://nanx.me/hdnom/appmaker" target="_blank">https://nanx.me/hdnom/appmaker</a>.
          The hdnom project website: <a href="https://nanx.me/hdnom/" target="_blank">https://nanx.me/hdnom/</a>.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>bioRxiv (2016)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="ivs"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>A Selective Review and Comparison for Interval Variable Selection in Spectroscopic Modeling</h3>

        <p>
          Li-Li Wang, You-Wu Lin, Xu-Fei Wang, <author>Nan Xiao</author>, Yuan-Da Xu, Hong-Dong Li, Qing-Song Xu.
        </p>

        <p>
          <img src="../images/paper-ivs.png" class="img-fluid rounded mx-auto d-block" alt="interval variable selection methods" width="80%" />
        </p>

        <p>
          <strong>Abstract:</strong> Dimension reduction and variable selection are two types of effective methods
          that deal with high-dimensional data. In particular, variable selection techniques are of wide-spread
          use and essentially consist of individual selection methods and interval selection methods.
          Given the fact that the vibrational spectra have continuous features of spectral bands,
          interval selection instead of individual spectral wavelength point selection allows for
          more stable models and easier interpretation. Numerous methods have been suggested for
          interval selection recently. Therefore, this paper is devoted to a selective review on
          interval selection methods with partial least squares (PLS) as the calibration model.
          We described the algorithms in the five classes: classic methods, penalty-based, sampling-based,
          correlation-based, and projection-based methods. Finally, we compared and discussed the
          performances of a subset of these methods on three real-world spectroscopic datasets.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Chemometrics and Intelligent Laboratory Systems (2018)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="ohpl"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>Ordered Homogeneity Pursuit Lasso for Group Variable Selection with Applications to Spectroscopic Data</h3>

        <p>
          You-Wu Lin, <author>Nan Xiao</author>, Li-Li Wang, Chuan-Quan Li, Qing-Song Xu.
        </p>

        <p>
          <img src="../images/paper-OHPL.png" class="img-fluid rounded mx-auto d-block" alt="OHPL algorithm flowchart" width="90%" />
        </p>

        <p>
          <strong>Abstract:</strong> In high-dimensional data modeling, variable selection methods have been
          a popular choice to improve the prediction accuracy by effectively selecting the subset of
          informative variables, and such methods can enhance the model interpretability with sparse representation.
          In this study, we propose a novel group variable selection method named ordered homogeneity pursuit lasso
          (OHPL) that takes the homogeneity structure in high-dimensional data into account. OHPL is particularly
          useful in high-dimensional datasets with strongly correlated variables. We illustrate the approach
          using three real-world spectroscopic datasets and compare it with four state-of-the-art variable
          selection methods. The benchmark results on real-world data show that the proposed method is
          capable of identifying a small number of influential groups and has better prediction performance
          than its competitors. The OHPL method and the spectroscopic datasets are implemented and included
          in an R package OHPL available from <a href="https://ohpl.io">https://OHPL.io</a>.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Chemometrics and Intelligent Laboratory Systems (2017)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="mrknn"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>A Novel Local Manifold-Ranking Based <em>k</em>-NN for Modeling the Regression Between Bioactivity and Molecular Descriptors</h3>

        <p>
          L. Shen, D.-S. Cao, Q.-S. Xu, X. Huang, <author>Nan Xiao</author>, Y.-Z. Liang.
        </p>

        <p>
          <img src="../images/paper-mrknn.png" class="img-fluid rounded mx-auto d-block" alt="manifold ranking k-nn" width="100%" />
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Chemometrics and Intelligent Laboratory Systems (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="msaenet"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>Multi-Step Adaptive Elastic-Net: Reducing False Positives in High-Dimensional Variable Selection</h3>

        <p>
          <author>Nan Xiao</author> and Q.-S. Xu.
        </p>

        <p>
          <img src="../images/paper-msaenet.png" class="img-fluid rounded mx-auto d-block" alt="msaenet" width="85%" />
        </p>

        <p>
          <strong>Abstract:</strong> Regression and variable selection in high-dimensional settings,
          especially when p >> n has been a popular research topic in statistical machine learning.
          In recent years, many successful methods have been developed to tackle this problem.
          In this paper, we propose the multi-step adaptive elastic-net (MSA-Enet), a multi-step
          estimation algorithm built upon adaptive elastic-net regularization. The numerical study
          on simulation data and real-world biological datasets have shown that the MSA-Enet method
          tends to significantly reduce the number of false-positive variables, while still maintain
          the estimation accuracy. By analyzing the variables eliminated in each step, more insight
          could be gained about the structure of the correlated variable groups. These properties
          are desirable in many real-world variable selection and regression problems.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Journal of Statistical Computation and Simulation (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="protr"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>protr/ProtrWeb: R Package and Web Server for Generating Various Numerical Representation Schemes of Protein Sequence</h3>

        <p>
          <author>Nan Xiao</author>, D.-S. Cao, M.-F. Zhu, and Q.-S. Xu.
        </p>

        <p>
          <img src="../images/paper-protr.png" class="img-fluid rounded mx-auto d-block" alt="protr" width="70%" />
        </p>

        <p>
          <strong>Abstract:</strong> Amino acid sequence-derived structural and physiochemical descriptors
          are extensively utilized for the research of structural, functional, expression and interaction
          profiles of proteins and peptides. We developed protr, a comprehensive R package for generating
          various numerical representation schemes of proteins and peptides from amino acid sequence.
          The package calculates eight descriptor groups composed of twenty two types of commonly used
          descriptors that include about 22,700 descriptor values. It allows users to select amino acid
          properties from the AAindex database, and use self-defined properties to construct customized
          descriptors. For proteochemometric modeling, it calculates six types of scales-based descriptors
          derived by various dimensionality reduction methods. The protr package also integrates the
          functionality of similarity score computation derived by protein sequence alignment and
          Gene Ontology (GO) semantic similarity measures within a list of proteins, and calculates
          profile-based protein features based on position-specific scoring matrix (PSSM).
          We also developed ProtrWeb, a user-friendly web server for calculating descriptors presented
          in the protr package. The protr package is freely available from
          <a href="https://cran.r-project.org/package=protr">CRAN</a>.
          ProtrWeb is freely available at <a href="http://protr.org">protr.org</a>.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Bioinformatics (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="Rcpi"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>Rcpi: R/Bioconductor Package to Generate Various Descriptors of Proteins, Compounds, and Their Interactions</h3>

        <p>
          D.-S. Cao*, <author>Nan Xiao*</author>, Q.-S. Xu and A. F. Chen. *Joint first authors.
        </p>

        <p>
          <img src="../images/paper-Rcpi.png" class="img-fluid rounded mx-auto d-block" alt="Rcpi" width="60%" />
        </p>

        <p>
          <strong>Abstract:</strong> In chemoinformatics and bioinformatics fields, one of the main computational
          challenges in various predictive modeling is to find a suitable way to effectively represent the molecules
          under investigation such as small molecules, proteins and even complex interactions. To solve this problem,
          we developed a freely available R/Bioconductor package, called Compound-Protein Interaction with R (Rcpi),
          for complex molecular representation from drugs, proteins and more complex interactions including
          protein-protein and compound-protein interactions. Rcpi could calculate a large number of structural
          and physicochemical features of proteins and peptides from amino acid sequences, molecular descriptors
          of small molecules from their topology, and protein-protein interaction and compound-protein interaction
          descriptors. In addition to main functionalities, Rcpi could also provide a number of useful auxiliary
          utilities to facilitate the user's need. With the descriptors calculated by this package, the users could
          conveniently apply various statistical machine learning methods in R to solve various biological and drug
          research questions in computational biology and drug discovery. Rcpi is freely available from the
          <a href="https://bioconductor.org/packages/Rcpi">Bioconductor website</a>.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Bioinformatics (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="MEF"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3>Integrating Multiple Evidence Sources to Predict Adverse Drug Reactions Based on Systems Pharmacology Model</h3>

        <p>
          D.-S. Cao, <author>Nan Xiao</author>, Y.-J. Li, W.-B. Zeng, Y.-Z. Liang, A.-P. Lu, Q.-S. Xu, A. F. Chen.
        </p>

        <p>
          <img src="../images/paper-mef.png" class="img-fluid rounded mx-auto d-block" alt="MEF" width="60%" />
        </p>

        <p>
          <strong>Abstract:</strong> Identifying potential adverse drug reactions (ADRs) is critically important for
          drug discovery and public health. Here we developed a multiple evidence fusion (MEF) method for the large-scale
          prediction of drug ADRs that can handle both approved drugs and novel molecules. MEF is based on the similarity
          reference by collaborative filtering, and integrates multiple similarity measures from various data types,
          taking advantage of the complementarity in the data. We used MEF to integrate drug-related and ADR-related
          data from multiple levels, including the network structural data formed by known drug–ADR relationships
          for predicting likely unknown ADRs. On cross-validation, it obtains high sensitivity and specificity,
          substantially outperforming existing methods that utilize single or a few data types. We validated our
          prediction by their overlap with drug–ADR associations that are known in databases. The proposed
          computational method could be used for complementary hypothesis generation and rapid analysis of
          potential drug–ADR interactions.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>CPT: Pharmacometrics & Systems Pharmacology (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class=""><div class="container"><div class="row"><div class="col-md-10 offset-1"><hr></div></div></div></div>

<a class="anchor" id="logd"></a>

<div class="py-3">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">

        <h3><em>In silico</em> Evaluation of logD<sub>7.4</sub> and Comparison with Other Prediction Methods</h3>

        <p>
          J.-B. Wang, D.-S. Cao, M.-F. Zhu, Y.-H. Yun, <author>Nan Xiao</author>, Y.-Z. Liang.
        </p>

        <p>
          <img src="../images/paper-logd.png" class="img-fluid rounded mx-auto d-block" alt="logd corrgram" width="50%" />
        </p>

        <p>
          <strong>Abstract:</strong> Lipophilicity, evaluated by either <i>n</i>-octanol/water partition coefficient (logP)
          or <i>n</i>-octanol/buffer solution distribution coefficient (logD), is of high importance in pharmacology,
          toxicology and medicinal chemistry. A quantitative structure-property relationship (QSPR) study was
          carried out for the prediction of distribution coefficients at pH 7.4 (logD<sub>7.4</sub>) of a large data set
          consisting of 1,130 organic compounds with 30 molecular descriptors selected by genetic algorithm (GA).
          Partial least squares (PLS) and support vector machine (SVM) regression were used to build prediction
          models with 904 molecules as the training set, and the predictive ability was evaluated with 226 molecules
          as the external test set. The results exhibited by the regression statistics demonstrate that the SVM model
          is more reliable and has a better predictive accuracy than the PLS model. The square correlation coefficients
          of fitting, cross validation and prediction are 0.92, 0.90 and 0.90, respectively. The corresponding root
          mean square errors are 0.52, 0.59 and 0.56, respectively. The reliability and generalization ability of the
          model were assessed by applicability domain and Y-randomization test, and 30 selected molecular descriptors
          could give a reliable and direct interpretation to logD<sub>7.4</sub> to some extent. When compared with the
          logD<sub>7.4</sub> values calculated by five methods from Discovery Studio and ChemAxon, our SVM model shows
          superiority over them. The results indicate that our model is a reliable and promising method to
          evaluate logD<sub>7.4</sub>.
        </p>

        <div class="row">
          <div class="col-md-10">
            <em>Journal of Chemometrics (2015)</em>.
          </div>
          <div class="col-md-2" align="right">
            <a href="#top">top</a>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<div class="nanx-footer">
  <div class="container">
    <div class="row">
      <div class="col-md-10 offset-1">
        <hr>
        <p class="text-muted">
          &copy; 2018 <a href="https://nanx.me/">Nan Xiao</a>. Built with Bootstrap 4.
        <ul class="list-inline">
          <li class="list-inline-item"><a href="https://github.com/road2stat" target="_blank">GitHub</a></li>
          <li class="list-inline-item"><a href="https://twitter.com/road2stat" target="_blank">Twitter</a></li>
          <li class="list-inline-item"><a href="https://www.linkedin.com/in/road2stat" target="_blank">LinkedIn</a></li>
          <li class="list-inline-item"><a href="https://scholar.google.com/citations?user=BNizRecAAAAJ&hl=en" target="_blank">Google Scholar</a></li>
        </ul>
        </p>
      </div>
    </div>
  </div>
</div>

<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-66394392-1', 'auto');
    ga('send', 'pageview');
</script>

</body>

</html>
